dwave/cloud/api/resources.py

```diff
@@ -275,3 +276,39 @@ def cancel_problems(self, problem_ids: List[str]) -> \
         response = self.session.delete(path, json=problem_ids)
         rtype = get_type_hints(self.cancel_problems)['return']
         return parse_obj_as(rtype, response.json())
+
+
+class LeapAccount(ResourceBase):
+
+    resource_path = 'account/'
+    client_class = LeapAPIClient
+
+    # TODO: constrain accepted resource/response version, when
+    # media type defined for Leap API responses
```

Comment:
@charleswhchan, I'll create an internal ticket for this.

Version decorators look, for example, like this:

https://github.com/dwavesystems/dwave-cloud-client/blob/d1bf2ee65e05bb974a7a2ef402c5a2413d060ae5/dwave/cloud/api/resources.py#L105-L106